bitnami/postgresql/values.yaml

```diff
@@ -185,6 +185,15 @@ extraEnv: []
 ##
 # postgresqlExtendedConf:
 
+# Configure current cluster's master server to be the standby server in other cluster.
+# This will allow cross cluster replication and provide cross cluster high availability.
+# You will need to configure pgHbaConfiguration if you want to enable this feature with local cluster replication enabled.
+##
+masterAsStandBy:
+  enabled: false
+#  masterHost:
+#  masterPort:
```

Comment:
```suggestion
## Configure current cluster's master server to be the standby server in other cluster.
## This will allow cross cluster replication and provide cross cluster high availability.
## You will need to configure pgHbaConfiguration if you want to enable this feature with local cluster replication enabled.
##
masterAsStandBy:
  enabled: false
  #  masterHost:
  #  masterPort:
```